horovod/spark/lightning/remote.py

```diff
@@ -24,7 +24,7 @@
 import pytorch_lightning as pl
 from pytorch_lightning import Trainer, Callback
 from pytorch_lightning.callbacks.model_checkpoint import ModelCheckpoint
-from pytorch_lightning.loggers import TensorBoardLogger
+from pytorch_lightning.loggers import TensorBoardLogger, CometLogger
```

Comment:
I didn't see `CometLogger` is used anywhere.

Reply:
correct. let me fix this.